class-tgm-plugin-activation.php

```diff
@@ -18,8 +18,8 @@
  * Plugin Name: TGM Plugin Activation
  * Plugin URI:
  * Description: Plugin installation and activation for WordPress themes.
- * Version:     2.5.0
- * Author:      griffinjt, garyj, jrf
+ * Version:     2.5.0-RC
+ * Author:      Thomas Griffin, Gary Jones, Juliette Reinders Folmer
```

Comment:
This is WP Plugin repo compliant header and should contain the WP.org nicks for the authors, so please revert.